starter_scripts/exaslct_within_docker_container_without_container_build.sh

```diff
@@ -12,4 +20,5 @@ if [ -z "$FIND_IMAGE_LOCALLY" ]; then
   docker pull "$RUNNER_IMAGE_NAME"
 fi
 
-bash "$SCRIPT_DIR/exaslct_within_docker_container.sh" "$RUNNER_IMAGE_NAME" "${@}"
+[[ "$(uname)" = Darwin ]] && EXEC_SCRIPT=exaslct_within_docker_container_slim.sh || EXEC_SCRIPT=exaslct_within_docker_container.sh
```

Comment:
Not sure, if I like this construct. It is shorter than an if, but I feel it has more potential for errors.